Source/WebKit/UIProcess/playstation/WebProcessPoolPlayStation.cpp

```diff
@@ -32,11 +32,47 @@
 #include <wpe/wpe.h>
 #endif
 
+#if ENABLE(REMOTE_INSPECTOR)
+#include <JavaScriptCore/RemoteInspectorServer.h>
+#include <wtf/text/StringToIntegerConversion.h>
+#endif
+
 namespace WebKit {
 
+#if ENABLE(REMOTE_INSPECTOR)
+static std::optional<uint16_t> webInspectorServerPort()
+{
+    static std::optional<uint16_t> portNumber { std::nullopt };
+    static std::once_flag flag;
+    std::call_once(flag, [] {
+        char buffer[64];
+        if (!getenv_np("WebInspectorServerPort", buffer, sizeof(buffer)))
```

Comment:
As a side note, the environment variable  `WebInspectorServerPort` is already defined and used in MiniBrowser:
https://github.com/WebKit/WebKit/blob/main/Tools/MiniBrowser/playstation/main.cpp#LL53C16-L53C38

Reply:
NIT: can we make that into a shared `static constexpr` or something so that we can guarantee it's shared by both?